content/notifications/send-notifications/get-started/user-guide/_index.en.md

```diff
@@ -4,50 +4,9 @@ linktitle: User guide
 weight: 10
 description: "A user guide for the notification service"
 ---
-
-{{% notice info %}}
-We are working on providing you with new and updated guidelines on usage of Altinn Notification. 
-In the meantime existing guidelines for sending notifications through Altinn 2 also apply to Altinn Notfications. 
-
-
-[Please familiarize yourself with existing documentation and guidelines](https://altinn.github.io/docs/utviklingsguider/varsling/)
-{{% /notice %}}
-
-
-
-
-<!---
-{{% notice info %}}
-TODO: say something about "varsel" vs "melding" or "beskjed". Kan brukes til begge deler. 
-What is important to think about ? 
-{{% /notice %}}
- 
-### Communication with private persons
-[Kontakt- og reservasjonsregisteret](https://eid.difi.no/nb/kontakt-og-reservasjonsregisteret) 
-(the common contact register) is used to retrieve contact information when an organisation sends a notification 
-through Altinn to a private person without providing the contact details to use. 
-
-In using this register both Altinn, and the sender of the notification,
-must adhere to [eForvaltningsforskriften §29](https://lovdata.no/forskrift/2004-06-25-988/§29). 
-
-We have summarized some of the information here, but encourage you to familiarize yourself with 
-[Digitaliseringsdirektoratet's guidelines for using the register](https://samarbeid.digdir.no/kontaktregisteret/retningslinjer-bruk-av-kontakt-og-reservasjonsregisteret/143). 
-
-
-Altinn Notifications should be used to support the execution of your organisations public authority. Meaning the use 
-should be linked to your organisation's _social mission_ and not for
-commercial purposes, marketing, or any other use that can be considered as private law.
-
-Altinn Notifications should be used when the message communicated in the notification contains 
-information that is important for the subject to receive and/or is important for the sending organisation 
-that the recipient receives.
-
-### Communication with persons as an entity of an organisation 
-
-### What should a notification include
-Varselet bør inneholde:
-- beskrivelse av oppgaven,
-- konkret hva den gjelder,
-- hvem som er ansvarlig/kan løse oppgaven
-- informer gjerne også om frist for oppgaven
--->
+## Notification
+### SMS
```

Comment:
Maybe we could add a section about us supporting sending notifications outside of Norway, and that all provided numbers are validated against a general list of country codes? Notifications to 5 digit numbers are not supported. 

This section could then be followed by the sentence:  "Additional examples of valid and invalid mobile numbers are provided below." And then your text below here

Reply:
Well, technically we aren't validating against any list of country code. 🤔 We are validating with the library and I didn't check how the library does that, most probably with a list of country codes of course, but not sure though. 🤔

Reply:
The issue here seems to be you not checking what the library actually does before introducing the dependency rather than what we are going to document here...

Reply:
Good point! This was our( me & @tba76 ) point of discussion before using this library that this library works as blackbox and do not check against provided list of country codes. Also as the library isn't quite well documented, therefore our(me and Thomas) process was to do manual testing before using it. As it turns out the library is pretty smart and only then we added it to the solution with the RegEx. 🤔 Hereby later we agreed to use it without the RegEx.

Reply:
Not sure I follow here. Of course the library validates against a list of known country codes. We know that is is able to successfully parse numbers such as +4799999999 into country code and number parts. As the library is successfully able to reject numbers with +47 stat start with something other than 4 or 9, we also know that the lib successfully validates it as a Norwegian number and follows the Norwegian number rules. 
Here is an overview of the country codes and the rule sets that the library uses: https://github.com/google/libphonenumber/tree/master/resources/metadata